routes/index.js

```diff
@@ -1,6 +1,18 @@
+const categoriesRoutes = require('./categories');
+const contentsRoutes = require('./contents');
+const regionsRoutes = require('./regions');
 const usersRoutes = require('./users');
+const volunteeringEventsRoutes = require('./volunteering_events');
+const volunteeringOffersRoutes = require('./volunteering_offers');
+const volunteeringRequestsRoutes = require('./volunteering_requests');
 
 module.exports = (router) => {
   router.prefix('/v1');
```

Comment:
I don't familiar with how to work with api versions.
I recommend using just /api prefix or learn how to use api versions.

Reply:
I think /api prefix should be used when deploying  only. About the version prefix, I'm not sure but I think it's OK for now.

Reply:
There is no point to do so, if we dont work with api versions, we dont need to use 'v1'.
Changing later just point that we won't use api versions.

Remember that if we change the prefix to 'v2' then all the API's will be /v2,
I don't think it's the behavior of api versions

Reply:
OK, you are right. I'm removing it then, and if we need it I guess we can do it via k8s.